user/fd_test/fd_test.c

```diff
@@ -184,6 +184,7 @@ void test_open_path() {
   char too_long[500];
   memset(too_long, 'c', sizeof(too_long));
   too_long[sizeof(too_long) - 1] = 0;
+  too_long[0] = '/';
```

Comment:
Added forward slash so path would be relative (we are trying to test fail because name is too long)